ament_tools/verbs/build/cli.py

```diff
@@ -158,6 +163,18 @@ def print_topological_order(opts, packages):
         sys.exit("Package '{0}' specified with --end-with was not found."
                  .format(opts.end_with))
 
+    opts.skip_packages = opts.skip_packages or []
+    nonexistant_skip_packages = []
+    for skip_package in opts.skip_packages:
+        if skip_package not in package_names:
+            nonexistant_skip_packages.append(skip_package)
+        if skip_package == opts.only_package:
+            sys.exit("Cannot --skip-packages and --only-package the same package: '{0}'."
+                     .format(skip_package))
+    if nonexistant_skip_packages:
+            sys.exit("Packages [{0}] specified with --skip-packages were not found."
+                     .format(', '.join(nonexistant_skip_packages)))
```

Comment:
It is in the other options, I think it is better so it catches typos. A warning would probably be missed since it is at the beginning of the output.